manager/manager.go

```diff
@@ -779,21 +777,31 @@ func (m *Manager) rotateRootCAKEK(ctx context.Context, clusterID string) error {
 			return err
 		}
 		unencryptedKeyBlock := &pem.Block{
-			Type:    keyBlock.Type,
-			Bytes:   unencryptedKey,
-			Headers: keyBlock.Headers,
```

Comment:
The headers specify whether it's encrypted, and what algorithm, etc:

```
-----BEGIN EC PRIVATE KEY-----
Proc-Type: 4,ENCRYPTED
DEK-Info: AES-256-CBC,fd7d7379e8f7728e054c4d01b2b8f27c
...
```

If the headers were left, then `x509.IsEncrypedPEMBlock` would return true even though the data itself is unencrypted. 

It's probably safer if we strip out only those specific headers, instead of all headers, but in most cases, we generate the key and don't include any additional headers, and if a user provides us with a key with extra headers, the extra headers aren't used, and they can get the key back out except by decrypting and dumping the snapshot or WALs.